.github/workflows/release.yml

```
@@ -0,0 +1,132 @@
+      id-token: write
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+      - name: Set up date and time variable
+        run: echo "DATE_TIME=$(TZ='Asia/Tokyo' date '+%Y-%m-%d-%H-%M-%S')" >> $GITHUB_ENV
+      - name: Log in to Docker Hub # https://github.com/docker/login-action
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Log in to the Container registry # https://github.com/docker/login-action
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)